src/js/components/CheckBox/README.md

```diff
@@ -328,6 +328,17 @@ Defaults to
 undefined
 ```
 
+**checkBox.label.alignment**
+
+Weather the checkbox is aligned center or
```

Comment:
Suggestion for description to align with Box align description: How to align the checkbox and label.